ledger/sandbox/src/main/scala/com/digitalasset/platform/index/JdbcIndexer.scala

```diff
@@ -273,6 +273,30 @@ class JdbcIndexer private[index] (
         )
         ledgerDao.uploadLfPackages(uploadId, packages, externalOffset).map(_ => ())(DEC)
 
+      case PackageUploadEntryAccepted(participantId, submissionId) =>
+        ledgerDao
+          .storePackageUploadEntry(
+            headRef,
+            headRef + 1,
+            externalOffset,
+            participantId,
+            submissionId,
+            None,
+            "accept")
```

Comment:
To avoid possible inconsistencies, please switch to a typed accept/reject representation (or, alternatively, define constants for such magic strings).

Reply:
addressed now, and put the magic strings as value of the corresponding case classes